### ISSUE TITLE
Tab order

### ISSUE THREAD

**Issue by daFiter** [Esri/storymap-shortlist#86]
My tab order seems to be random and ever changing. I am trying to color the tabs based on the content but since I cannot get them in the order I want consistently I cannot assign a proper color. 

 I am using the "As-is" version. I have also hosted the app on my own server. 

I have looked at this: https://community.esri.com/thread/182899-how-are-points-in-a-shortlist-story-map-tab-ordered

and it says that "The order in which the tabs will be shown in the Shortlist is based on the actual record order of the TAB_NAME field" what does that mean? the Object id order? 

an guidance would be appreciated.

**Comment by RupertEssinger** [comment]
Hi Rickey
Sorry about the prob, 

Could you post the URL to your web map.

The order of your tabs is controlled by the actual record order of the TAB_NAME values. So it is the order that the table appears in if you open it in the ArcGIS Online Map Viewer. That order isn't controlled by a field in the table. You can permanently change the order of your table by doing a geoprocessing sort.

There is an issue with tab order and place order not working consistently when the 'as is' workflow is used with a layer published using ArcGIS Enterprise Server, so we don't recommend using that data source.

**Comment by daFiter** [comment]
Here is my URL

http://gis.ashland.or.us/cipstorybook/

**Comment by RupertEssinger** [comment]
Thanks, could we also get the URL of the web map your app is 'as-is'-ing to.

**Comment by daFiter** [comment]
http://ashlandgis.maps.arcgis.com/home/webmap/viewer.html?webmap=19e40d4c03d4477d915217711cc8fce7

**Comment by daFiter** [comment]
I used the sort tool and I now have the tabs in the right order. 
Thank you!

**Comment by RupertEssinger** [comment]
I get prompted to sign in when I open your web map. Could you share it and content publicly so we can do some testing too?

If your layer is based on an AGS Server services (rather than a hosted ArcGIS Online feature service) I would check to make sure the tabs and the places stay in the right order. We'd recommend using hosted feature services instead of AGS Server services for 'as-is' due to issues we have seen in Shortlist with how records are read in.
